cli/server/server.go

```diff
@@ -16,7 +16,7 @@ type appConfig struct {
 	Host        string `envconfig:"HOST"`
 	Port        int    `envconfig:"PORT" default:"8080"`
 	UIDomain    string `envconfig:"UI_DOMAIN" default:"http://127.0.0.1:8080"`
-	DBConn      string `envconfig:"DB_CONNECTION" default:"sqlite://./internal.db"`
+	DBConn      string `envconfig:"DB_CONNECTION" default:"sqlite:///var/code-annotation/internal.db"`
```

Comment:
by default server will fail because `/var/code-annotation` doesn't exist.
not very user-friendly.

Reply:
Since we're doing prod first that's the expected location we agreed.
We could maybe change the `.env.tpl` to be:
```shell
DB_CONNECTION=sqlite://internal.db
```

Reply:
that's what I mean. It's absolutely okay to have `/var/code-annotation` path as default, but let's keep it simple for developers. Thanks!

Reply:
👍  If anybody else wonders, feedback seems to be addressed and [default for dev env](https://github.com/src-d/code-annotation/pull/85/files#diff-4ea7d7d38652a9124d508ef1d16ac78dR5) was changed as requested.